hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -182,6 +182,7 @@
     public static final String PROP_LOGGING_TYPE = "hazelcast.logging.type";
     public static final String PROP_ENABLE_JMX = "hazelcast.jmx";
     public static final String PROP_ENABLE_JMX_DETAILED = "hazelcast.jmx.detailed";
+    public static final String PROP_JMX_INTERVAL = "hazelcast.jmx.interval";
```

Comment:
Maybe we should also add the time unit to the name, so it's clear if we have to set seconds or milliseconds here.